integrations/oci/oci-secrets-config-source/pom.xml

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2023 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.helidon.integrations.oci</groupId>
+        <artifactId>helidon-integrations-oci-project</artifactId>
+        <version>4.0.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>helidon-integrations-oci-secrets-config-source</artifactId>
+    <name>Helidon Config OCI Secrets ConfigSourceProvider</name>
+
+    <description>
+        OCI Secrets Retrieval API ConfigSourceProvider Implementation
+    </description>
+
+    <properties>
+        <compartment-ocid/>
+        <java.util.logging.config.file>src/test/java/logging.properties</java.util.logging.config.file>
+        <vault-ocid/>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <!--
+                OCI brings in two different versions of httpclient and httpcore (!); here we tell it to use the most
+                recent of the versions of each that it brings in.
+            -->
+            <dependency>
+                <groupId>org.apache.httpcomponents</groupId>
+                <artifactId>httpclient</artifactId>
+                <version>4.5.14</version>
```

Comment:
versions should be externalized to above properties or in dependencies/pom.xml - deferring to @barchetta

Reply:
Joe and I came up with this together. If you remove these, dependency convergence fails. What solution do the two of you propose? I am happy to follow directions.

Reply:
Replicated hack from `integrations/oci/cdi`. Done.